test_scripts/Defects/4_5/1880_DefaultAndRPCTimeoutGENERIC_ERROR.lua

```diff
@@ -0,0 +1,403 @@
+    {
+      text = "Make your choice",
+      type = "TEXT"
+    }
+  },
+  interactionChoiceSetIDList = {100},
+  timeout = 8000,
+  interactionLayout = "LIST_ONLY"
+}
+
+local PerformInteractionRequestParamsVR = {
+  initialText = "StartPerformInteraction",
+  initialPrompt = {
+    {
+      text = "Make your choice",
+      type = "TEXT"
+    }
+  },
+  interactionMode = "VR_ONLY",
+  interactionChoiceSetIDList = {100},
+  helpPrompt = {
+    {
+      text = "Help Prompt",
+      type = "TEXT",
+    }
+  },
+  timeoutPrompt = {
+    {
+      text = "Time out",
+      type = "TEXT",
+    }
+  },
+  timeout = 12000,
+  vrHelp = {
+    {
+      text = "New VRHelp",
+      position = 1,
+      image = {
+        value = "icon.png",
+        imageType = "STATIC",
+      }
+    }
+  },
+  interactionLayout = "ICON_ONLY"
+}
+
+--[[ Local Functions ]]
+local function Alert(params, self)
+  local AlertDuration
+  if params.duration then
+    AlertDuration = params.duration
+  else
+    -- deration default valur from mobile API
+    AlertDuration = 5000
+  end
+  local RespTimeout = DefaultTimeout + AlertDuration
+  local RequestTime
+  local RespTime
+  local TimeBetweenReqRes
+
+  if params.ttsChunks then
+    EXPECT_HMICALL("TTS.Speak")
+    :Do(function(_,data)
+        local function SpeakResp()
+          self.hmiConnection:SendResponse(data.id, data.method, "SUCCESS", {})
+        end
+        RUN_AFTER(SpeakResp, 2000)
+      end)
+  end
+
+  local cid = self.mobileSession1:SendRPC("Alert", params)
+  RequestTime = timestamp()
+
+  EXPECT_HMICALL("UI.Alert")
+
+  self.mobileSession1:ExpectResponse(cid, { success = false, resultCode = "GENERIC_ERROR" })
+  :Timeout(RespTimeout + 1000)
+  :ValidIf(function()
+      RespTime = timestamp()
+      TimeBetweenReqRes = RespTime - RequestTime
+      if
+      TimeBetweenReqRes > RespTimeout - 1000 and
+      TimeBetweenReqRes < RespTimeout + 1000 then
+        return true
+      else
+        return false, "SDL triggers timeout earlier then expected(".. tostring(RespTimeout) .." sec), after " ..
+        tostring(TimeBetweenReqRes) ..
+        " sec.\n SDL must use Alert duration + default timeout in case of absence softButtons."
+      end
+    end)
+end
+
+local function Slider(params, self)
+  local SliderDuration
+  if params.timeout then
+    SliderDuration = params.timeout
+  else
+    -- timeout default value from mobile API
+    SliderDuration = 10000
+  end
+  local RespTimeout = DefaultTimeout + SliderDuration
+  local RequestTime
+  local RespTime
+  local TimeBetweenReqRes
+  local cid = self.mobileSession1:SendRPC("Slider", params)
+  RequestTime = timestamp()
+  EXPECT_HMICALL("UI.Slider")
+  self.mobileSession1:ExpectResponse(cid, {success = false, resultCode = "GENERIC_ERROR"})
+  :Timeout(RespTimeout + 1000)
+  :ValidIf(function()
+      RespTime = timestamp()
+      TimeBetweenReqRes = RespTime - RequestTime
+      if
+      TimeBetweenReqRes > RespTimeout - 1000 and
```

Comment:
@GetmanetsIrina Please make similar indentations across the whole file

Reply:
@AByzhynar Done in fdecb8b